### PR TITLE
Remove docs from gis4wrf plug-in

### PR DESCRIPTION
**Pull request by dmey**
With the next release I would suggest to host all the docs and tutorial directly online and remove the offline docs altogether from the home tab. This will also remove dependencies on QWebPage +QWebView.  I have removed all the bloated code for generating the offline docs too.  Can you please have a look/try and let me know what you think? Thanks

### CODE REVIEW

File: gis4wrf/plugin/ui/tab_home.py [hunk 1]
```diff
@@ -1,16 +1,51 @@
...
 
-from PyQt5.QtWidgets import QWidget
 from pathlib import Path
 
-from gis4wrf.plugin.ui.helpers import create_browser_layout
 
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QWidget, QVBoxLayout, QLabel
+from PyQt5.QtGui import QPixmap
+from gis4wrf.plugin.ui.helpers import FormattedLabel
```
Comment (letmaik) [inline gis4wrf/plugin/ui/tab_home.py]: blank line between external/internal imports

File: gis4wrf/plugin/ui/tab_home.py [hunk 2]
```diff
@@ -1,16 +1,51 @@
...
+                        </font>
+                  </html>
+               """
+
+        label_title = FormattedLabel(title, align=True)
+        label_text = FormattedLabel(text, align=True)
+        label_text.setWordWrap(True)
+        label_text.setOpenExternalLinks(True)
+        label_pixmap = QLabel()
+        pixmap = QPixmap(GIS4WRF_LOGO_PATH)
```
Comment (letmaik) [inline gis4wrf/plugin/ui/tab_home.py]: 👍

File: gis4wrf/plugin/ui/tab_home.py [hunk 3]
```diff
@@ -1,16 +1,51 @@
...
+                    <html>
+                        <font size="4">
+                        <br>
+                        <p>The GIS4WRF documentation and tutorials have been moved online at: <a href="https://gis4wrf.github.io">https://gis4wrf.github.io</a></p>
+                        <br>
+                        <p>We are delighted to announce that we can now provide MPI-enabled WPS-V4 and WRF-V4 pre-built binaries for Windows, macOS and Linux through WRF-CMake. 😊</p>
+                        <p>If you have not done so already, make sure to download the latest V4 releases for your system.
+                            For more info see: <a href="https://gis4wrf.github.io/configuration">https://gis4wrf.github.io/configuration</a></p>
+                        <br>
+                        <p>Make sure to check out all the new features with this version under <code>GIS4WRF</code> > <code>About</code> > <code>What's new</code></p>
```
Comment (letmaik) [inline gis4wrf/plugin/ui/tab_home.py]: misses "Plugins ->"?

File: gis4wrf/plugin/ui/tab_home.py [hunk 4]
```diff
@@ -1,16 +1,51 @@
...
+                """
+        text = """
+                    <html>
+                        <font size="4">
+                        <br>
+                        <p>The GIS4WRF documentation and tutorials have been moved online at: <a href="https://gis4wrf.github.io">https://gis4wrf.github.io</a></p>
+                        <br>
+                        <p>We are delighted to announce that we can now provide MPI-enabled WPS-V4 and WRF-V4 pre-built binaries for Windows, macOS and Linux through WRF-CMake. 😊</p>
+                        <p>If you have not done so already, make sure to download the latest V4 releases for your system.
+                            For more info see: <a href="https://gis4wrf.github.io/configuration">https://gis4wrf.github.io/configuration</a></p>
```
Comment (letmaik) [inline gis4wrf/plugin/ui/tab_home.py]: Is it normal that I can't click any of the links?

**Comment by dmey** [comment]
I have just changed the font size, can you let me know if it still looks OK on yours?

File: gis4wrf/plugin/ui/tab_home.py (hunk 4 above)
Comment (letmaik) [reply gis4wrf/plugin/ui/tab_home.py]: Fixed it, had to do with the FormattedLabel class which wasn't used in the other cases, dead code, and this breaks hyperlinks for some reason.

**Comment by letmaik** [comment]
It's a bit small now:

image

**Comment by letmaik** [comment]
It's the same size as all the other UI text, yes, but the heading is so huge that it feels super tiny. And I really hate "websizes" which have tiny text for their articles. Why not bigger?

**Comment by dmey** [comment]
I still cannot tell how big it is relative to you maximised (full screen) window.  On both Windows and macOS Retina looks OK

image

screen shot 2018-09-17 at 21 54 29

**Comment by letmaik** [comment]
You have a small laptop screen, that's why there's not much white space. But looks like mac scaling works better in general in qgis.

image

**Comment by dmey** [comment]
I tested it on a 16:9 HD screen and it looks the same as the pic I posted so I think it's fine for now.

**Pull request merged**